apm-sniffer/apm-sdk-plugin/spring-plugins/spring-webflux-5.x-webclient-plugin/src/main/java/org/apache/skywalking/apm/plugin/spring/webflux/v5/webclient/WebFluxWebClientInterceptor.java

```diff
@@ -79,15 +78,12 @@ public Object afterMethod(EnhancedInstance objInst, Method method, Object[] allA
         }
         Mono<ClientResponse> ret1 = (Mono<ClientResponse>) ret;
         AbstractSpan span = (AbstractSpan) context.getContext();
-        return ret1.doAfterSuccessOrError(new BiConsumer<ClientResponse, Throwable>() {
-            @Override
-            public void accept(ClientResponse clientResponse, Throwable throwable) {
-                HttpStatus httpStatus = clientResponse.statusCode();
-                if (httpStatus != null) {
-                    Tags.STATUS_CODE.set(span, Integer.toString(httpStatus.value()));
-                    if (httpStatus.isError()) {
-                        span.errorOccurred();
-                    }
+        return ret1.doOnSuccess(clientResponse -> {
```

Comment:
So, even the response code is 5xx, this method would be called?

Reply:
Yes, this method will be called unless throw an exception.